games-action/cylindrix/cylindrix-1.0

```diff
@@ -1,15 +1,16 @@
 SUMMARY="A combat game"
 DESCRIPTION="Cylindrix is a 3 degrees of freedom combat game"
 HOMEPAGE="https://www.allegro.cc/depot/Cylindrix"
-COPYRIGHT="2001 John R. McCawley III, Anthony Thibault"
+COPYRIGHT="2001 John R. McCawley III
+	2001 Anthony Thibault"
 LICENSE="GNU GPL v2"
 REVISION="1"
 SOURCE_URI="https://d1cxvcw9gjxu2x.cloudfront.net/projects/1235#cylindrix-1.0.tar.gz"
 CHECKSUM_SHA256="33bfc264d906a1977090e47a38f544551d2939d4c0ea01e1792f63ab4fa06f20"
 SOURCE_DIR="cylindrix"
 
-ARCHITECTURES="!x86_gcc2 !x86 !x86_64"
-SECONDARY_ARCHITECTURES="!x86"
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
```

Comment:
Don't set the architecture as working before it actually runs the game
```diff
-ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+ARCHITECTURES="?x86_gcc2 ?x86 ?x86_64"
```

Reply:
![image](https://user-images.githubusercontent.com/30193966/71576611-d4955080-2af9-11ea-8d87-b539d57525e7.png)

Will change, it does launch the game, but it does get stuck in the splash screen and it also seems to be buggy.

Reply:
Already been at that point in the past, hence it was disabled :)

Reply:
> Already been at that point in the past, hence it was disabled :)

So fixing the COPYRIGHT section seems to be enough-- I'll invert the changes.